requirements.txt

```diff
@@ -19,4 +19,6 @@ flask-jwt-extended==3.25.1
 sphinx
 rinohtype
 celery~=5.0.5
-redis==3.5.3
\ No newline at end of file
+redis==3.5.3
+psycopg2
+boto3
```

Comment:
it would be good to tag the majo version atleast to this SDK `boto3==x` or inclusive the minor tag too `boto3==x.x`

that way we leave room for minor fixes which comes in the 3rd tag mostly and still not risk breaking of the module usage due to upgrades and locations of objects imported

Reply:
Okay noted